### PR TITLE
pythonPackages.mbed-ls: init at 1.6.2

### PR DESCRIPTION
**Pull request by rvolosatovs**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/python-modules/mbed-ls/default.nix [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  propagatedBuildInputs = [
+    appdirs
+    fasteners
+    prettytable
+  ];
+
+  doCheck = false; # tests fail
+
+  meta = with lib; {
+    description = "mbed-ls is a Python module that detects and lists mbed-enabled devices connected to the host computer";
```
Comment (dotlambda) [inline pkgs/development/python-modules/mbed-ls/default.nix]: https://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes

File: pkgs/development/python-modules/mbed-ls/default.nix [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    sha256 = "0nyb3cw4851cs8201q2fkna0z565j7169vj7wm2c88c8fm6qd21i";
+  };
+
+  propagatedBuildInputs = [
+    appdirs
+    fasteners
+    prettytable
+  ];
+
+  doCheck = false; # tests fail
```
Comment (dotlambda) [inline pkgs/development/python-modules/mbed-ls/default.nix]: You should disable specific tests.

**Review by dotlambda — CHANGES_REQUESTED** [review]

File: pkgs/top-level/python-packages.nix [hunk 3]
```diff
@@ -2981,6 +2981,8 @@ in {
 
   maya = callPackage ../development/python-modules/maya { };
 
+  mbed-ls = callPackage ../development/python-modules/mbed-ls {  };
```
Comment (dotlambda) [inline pkgs/top-level/python-packages.nix]:
```suggestion
  mbed-ls = callPackage ../development/python-modules/mbed-ls { };
```

**Review by dotlambda — CHANGES_REQUESTED** [review]

File: pkgs/development/python-modules/mbed-ls/default.nix [hunk 4]
```diff
@@ -0,0 +1,26 @@
+{ lib , buildPythonPackage , fetchPypi, appdirs, fasteners, prettytable }:
```
Comment (dotlambda) [inline pkgs/development/python-modules/mbed-ls/default.nix]:
```suggestion
{ lib, buildPythonPackage, fetchPypi, appdirs, fasteners, prettytable }:
```

**Review by dotlambda — CHANGES_REQUESTED** [review]

**Comment by rvolosatovs** [comment]
@dotlambda

**Comment by dotlambda** [comment]
@GrahamcOfBorg build python2.pkgs.mbed-ls python3.pkgs.mbed-ls

**Comment by dotlambda** [comment]
You might want to report the test failure on aarch64 to upstream.

**Comment by rvolosatovs** [comment]
@dotlambda

**Comment by dotlambda** [comment]
@GrahamcOfBorg build python2.pkgs.mbed-ls python3.pkgs.mbed-ls

**Comment by dotlambda** [comment]
> You might want to report the test failure on aarch64 to upstream.

@rvolosatovs Did you do that? We should probably set `meta.broken = stdenv.isAarch64` and add a comment with a link to the issue.

**Comment by rvolosatovs** [comment]
The package is deprecated, closing https://github.com/ARMmbed/mbed-os-tools/tree/master/packages/mbed-ls#development-moved

**Pull request closed without merging**